iac/tf-multienv-cicd-anthos-autopilot/shell-scripts/03-wait-for-asm.sh

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -Eeuo pipefail
+
+echo "🚀  Starting $0"
+echo '🕰  Waiting for GKE cluster setup to complete provisioning managed Service Mesh with managed Control Plane and managed Data Plane.'
```

Comment:
TF doesn't do that waiting? :(

Reply:
TF is completely done after 02-terraform.sh

As these scripts' purpose is mainly to make it easier for me to test reproducibility and it seems they are source for quite a bit of confusion, I'm removing the entire script folder. The necessary one-time setup steps will instead be moved into README.md.